Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -300,6 +340,10 @@ export interface QTestArguments extends Transformer.RunnerArguments {
     weight?: number;
     /** Privilege level required by this process to execute. */
     privilegeLevel?: "standard" | "admin";
+     /** Directory to write coverage logs in */
+    coverageLogsDir?: Directory;

```

Comment:
Does this need to be configurable? How about just making it a child directory of qTestLogs or put it in the qtest sandbox directory?

Reply:
In DScript, what is the right way to create a subdirectory of qTestLogs and pass it as an input to the coverage pip? I tried
```const covdataDir: StaticDirectory = result.getOutputDirectory(d`${qTestLogsDir}/CovData`);```
and received the value "undefined" for covdataDir.

Reply:
It should be something like this:
`const covDataDir: Directory = d`${logDir}\CovData`;`

Reply:
I'd tried this, but BuildXL then complains that I'm trying to write to a sealed directory.

Reply:
I'd tried this as well, but then BuildXL throws an error saying that the content of a fully-sealed directory can no longer change.

Reply:
That would happen if the directory or a parent directory had previously been "sealed". Usually that only happens via Transformer.sealDirectory(), but the parent directory is an opaque directory which has an implicit seal since opaque directories are exclusive for a single pip. In other words making it a child isn't going to work. Maybe just make it a sibling to logDir, perhaps just add _CovData suffix on it. My main point here is that it seems unnecessary to have so many configurable options since the covdata location is mostly an internal detail.